### PR TITLE
Hotfix to support local docs with a correct BokehJS

### PR DESCRIPTION
**Pull request by damianavila**
This just add an entry point using an env variable to build the docs and get the plot rendered when you are working locally. This let's you use any released BokehJS or any devel version uploaded to the CDN (there could be the case, although not frequently, that you need a newer version that the released one and then, the devel version makes more sense).

### CODE REVIEW

File: bokeh/resources.py [hunk 1]
```diff
@@ -32,7 +32,10 @@ def _cdn_base_url():
 
 def _get_cdn_urls(version=None, minified=True):
     if version is None:
-        version = __version__.split('-')[0]
+        if settings.local_docs_cdn:
```
Comment (bryevdv) [inline bokeh/resources.py]: missing parens :)

File: sphinx/source/docs/dev_guide.rst [hunk 2]
```diff
@@ -631,6 +631,9 @@ There are several environment variables that can be useful for developers:
 * ``BOKEH_VERSION`` --- What version of BokehJS to use with ``cdn`` resources
     See the :class:`~bokeh.resources.Resources` class reference for full details.
 
+* ``BOKEH_LOCAL_DOCS_CDN`` --- What version of BokehJS to use when you are working
+    with sphinx docs locally.
+
```
Comment (bryevdv) [inline sphinx/source/docs/dev_guide.rst]: This will conflict with dev guide PR, just FYI

File: bokeh/resources.py (hunk 1 above)
Comment (damianavila) [inline bokeh/resources.py]: jaja, to late here... thanks

File: sphinx/source/docs/dev_guide.rst (hunk 2 above)
Comment (damianavila) [inline sphinx/source/docs/dev_guide.rst]: yep, just throwing the idea to get feedback...

**Comment by bryevdv** [comment]
LGTM, merging

**Comment by bryevdv** [comment]
(I will handle the merge conflict in the other PR)

**Comment by damianavila** [comment]
> (I will handle the merge conflict in the other PR)

Thanks!

**Pull request merged**